beacon-chain/rpc/eth/light-client/handlers.go

```diff
@@ -46,18 +47,62 @@ func (s *Server) GetLightClientBootstrap(w http.ResponseWriter, req *http.Reques
 		return
 	}
 
-	bootstrap, err := createLightClientBootstrap(ctx, state)
-	if err != nil {
-		httputil.HandleError(w, "could not get light client bootstrap: "+err.Error(), http.StatusInternalServerError)
+	switch blk.Version() {
+	case version.Phase0:
+		httputil.HandleError(w, "light client bootstrap is not supported for phase0", http.StatusNotImplemented)
```

Comment:
The HTTP code should be `http.BadRequest`. `StatusNotImplemented` means that the server doesn't have some functionality implemented, but there is no functionality to implement here.